cypress/elements/optionsModal/verticalAxis.js

```diff
@@ -0,0 +1,50 @@
+const titleCheckboxEl = 'option-vertical-axis-title-checkbox'
+const titleInputEl = 'option-vertical-axis-title-input'
+const rangeMinInputEl = 'option-vertical-axis-range-min-input'
+const rangeMaxInputEl = 'option-vertical-axis-range-max-input'
+
+export const enableVerticalAxisTitle = () =>
+    cy
+        .getBySel(titleCheckboxEl)
+        .find('[type="checkbox"]')
+        .check({ force: true })
+        .should('be.checked')
```

Comment:
Is this function actually an expectation? Then it should be renamed as such. But it seems like an action function, in which case I would argue that the `should` should be dropped.

Reply:
This function is a workaround to check the `dhis2/ui` `checkbox` components, because they're custom checkboxes which Cypress can't "see", which is why the `force: true` is used. However, to verify that the checkbox was actually checked (since it could be hidden for another reason) I added the should to be sure. I'll add a `FIXME` comment to find another way to check the checkbox though.